### PR TITLE
AboutCreated

### PR DESCRIPTION
**Pull request by AytenZeynalova**

### CODE REVIEW

File: src/pages/AboutUsPage/AboutUsPage.scss [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+.line{
+    background-color: rgb(234, 222, 215);
+    min-height: 20vh;
+    display: flex;
+    align-items: center;
+    
+    h3{
+        font-size: 1.5rem;
+        color: #453227;
```
Comment (Negodyaj) [inline src/pages/AboutUsPage/AboutUsPage.scss]: you must use variables for colors

File: src/pages/AboutUsPage/AboutUsPage.scss [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        letter-spacing: 0.1rem;
+        margin-left: 0% !important;
```
Comment (Negodyaj) [inline src/pages/AboutUsPage/AboutUsPage.scss:13]: why do you need this !important?..

File: src/pages/AboutUsPage/AboutUsPage.scss [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    letter-spacing: 0.1rem;
+    margin-bottom: 0.75rem;
+    color: #102a42;
+  }
+  .underline {
+    margin-left: 0;
+    height: 0.25rem;
+    width: 6rem;
+    margin-right: auto;
+    background-color: #ab7a5f;
```
Comment (Negodyaj) [inline src/pages/AboutUsPage/AboutUsPage.scss]: please use variables for all the colors in this file

Comment (AytenZeynalova) [reply src/pages/AboutUsPage/AboutUsPage.scss]: okay i will use

**Pull request merged**